cognite/pygen/_core/templates/data_class_edge.py.jinja

```diff
@@ -343,7 +343,7 @@ class _{{ data_class.query_cls_name }}(EdgeQueryCore[T_DomainList, {{ data_class
                 ),
                 {% elif field.is_reverse_direct_relation %}
                 dm.query.NodeResultSetExpression(
-                    through={{ field.reverse_property.data_class.view_id_str }}.as_property_ref("{{ field.prop_name }}"),
+                    through={{ field.through_str }},
```

Comment:
Maybe stupid question: where is this through_str coming from?

Reply:
It is part of the internal model pygen uses, https://github.com/cognitedata/pygen/blob/main/cognite/pygen/_core/models/fields/connections.py#L228. 

Note pygen works by taking views -> creating an internal model -> use the internal model to create the SDK.

Reply:
Issue in this PR was simply that this edge case (reverse direct relation on an edge property) was not tested.